tests/test_orchestrator.py

```diff
@@ -37,12 +37,12 @@ def test_inactive_orc_get_address():
         db.get_address()
 
 
-def test_orc_active_functions(fileutils):
+def test_orc_active_functions(fileutils, wlmutils):
     exp_name = "test_orc_active_functions"
     exp = Experiment(exp_name, launcher="local")
     test_dir = fileutils.make_test_dir()
 
-    db = Orchestrator(port=6780)
+    db = Orchestrator(port=wlmutils.get_test_port())
```

Comment:
This is OK, but at line 54 we test that the db is running at the correct address. That way was OK as long as we were using a fixed port, can you update the `assert` statement so that it uses the right port?

Reply:
Also, there is a problem with this test, if the orchestrator fails to launch, we don't shut it down. It would be great if you could fix this as part of this ticket. 
We implemented a couple workarounds for cases like this in other tests or you could possibly also use the `local_db` fixture... I leave it up to you!